crds/virtualmachineclasses.yaml

```diff
@@ -2,9 +2,10 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
   name: virtualmachineclasses.virtualization.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: virtualization
```

Comment:
crd vmcalss is automatically generated. Your changes will be replaced next time

Reply:
@yaroslavborbat so mb add these labels to generator?